src/pages/index.astro

```diff
@@ -25,7 +25,7 @@ const featuredContracts = [
 <Layout>
   <div class="mycelium text-center">
     <div class="text-left">
-      <Logo />
+      <wc-logo></wc-logo>
```

Comment:
No need to `import`, can use the selected name for the component (in this case `wc` stands for **w**eb-**c**omponent, we can go with any other convention).